tests/webhooks/test_e2e.py

```diff
@@ -99,6 +99,21 @@ def get_logs(self) -> list[WebhookLogEntry]:
         logs_data = response.json().get("logs", [])
         return [WebhookLogEntry(**log_data) for log_data in logs_data]
 
+    def wait_for_logs(self, expected_count: int, timeout: float = 2.0) -> list[WebhookLogEntry]:
```

Comment:
nit: do we need to accept timeout as an argument? This does not seem to be used

Reply:
Yes, I need to use this test request retries.